EIPS/eip-2611.md

```diff
@@ -0,0 +1,152 @@
+---
+eip: 2611
```

Comment:
Where is 2611 coming from? That seems to be something different: https://github.com/ethereum/EIPs/pull/2611

Reply:
Got it - thank you. Updated.